wbooks/src/app/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import logoWolox from './assets/LogoWolox.png';
+import './application.css';
+
+class App extends Component {
+  state = {
+    name: 'Nombre',
+    lastname: 'Apellido',
+    email: 'example@domain.com',
+    password: 'password',
+    password_confirm: 'password'
+  };
+
+  render () {
+    return (
+      <React.Fragment>
+        <div class='sign-up-container'>
+          <img class='wolox-logo' src={logoWolox} alt=''/>
+          <form onSubmit={this.handleSubmit} class='sign-up-form' method="post">
+            <label class='sign-up-label'>Nombre</label>
```

Comment:
All the fields are very similar except for the values.  You can use `map` here or create a `Component`